client/packages/system/src/Stock/Components/Repack/RepackModal.tsx

```diff
@@ -137,7 +137,12 @@ export const RepackModal: FC<RepackModalControlProps> = ({
       okButton={
         <DialogButton
           variant="save"
-          disabled={draft?.newPackSize === 0 || draft?.numberOfPacks === 0}
+          disabled={
+            draft?.newPackSize === 0 ||
+            draft?.numberOfPacks === 0 ||
+            !draft.newPackSize ||
+            !draft.numberOfPacks
+          }
```

Comment:
I really wanted to just approve because I'm sure this works but... can't we just use that `0` is falsy in JS?
```suggestion
          disabled={!draft?.newPackSize || !draft?.numberOfPacks}
```

Easy to ignore but after a while I think it kinda bloats the code base. On the fence of when to request changes...

Reply:
Done~